batch_util_example.py

```diff
@@ -14,52 +14,69 @@
 from datetime import date, timedelta
 import demisto
 
+
 def format_dt(dt):
     return dt.strftime('%Y-%m-%dT%H:%M:%S')
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Utility for batch action on incidents')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
+    parser = argparse.ArgumentParser(
+        description='Utility for batch action on incidents')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
     monthAgo = date.today() - timedelta(days=30)
-    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is open incidents created in last month', default='(status:=0 or status:=1) and created:>%s' % format_dt(monthAgo))
-    parser.add_argument('-m', '--page', help='The page we are working on', default=0, type=int)
-    parser.add_argument('-n', '--size', help='The size per page', default=100, type=int)
-    parser.add_argument('-a', '--action', help='The action to perform', default='close', choices=['close'])
+    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is open incidents created in last month',
+                        default='(status:=0 or status:=1) and created:>%s' % format_dt(monthAgo))
+    parser.add_argument(
+        '-m', '--page', help='The page we are working on', default=0, type=int)
+    parser.add_argument(
+        '-n', '--size', help='The size per page', default=100, type=int)
+    parser.add_argument('-a', '--action', help='The action to perform',
+                        default='close', choices=['close'])
     parser.add_argument('--closeReason', help='The close reason')
     parser.add_argument('--closeNotes', help='The close notes')
-    parser.add_argument('--customFields', help='The json that includes the values for the custom fields')
+    parser.add_argument(
+        '--customFields', help='The json that includes the values for the custom fields')
     parser.add_argument('--playbook', help='The new playbook name')
     parser.add_argument('--type', help='The new type')
     parser.add_argument('--entry', help='The new entry data to create')
-    parser.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True, help="don't use voice")
+    parser.add_argument('-q', '--quiet', action='store_false',
+                        dest='verbose', default=True, help="don't use voice")
```

Comment:
`default` is automatically set to `True` when `action='store_false'` is set.

Reply:
removed default option from where store_true of store_false is set.(e44c2f8)